test/hooks-order.js

```diff
@@ -0,0 +1,277 @@
+
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import {outdent} from 'outdent';
```

Comment:
Why bother with a new dependency ? Additional indent is not an issue here.

Reply:
It's not necessary. I was copying a pattern from a different repo.